.devcontainer/Dockerfile

```diff
@@ -1,16 +1,58 @@
-FROM ghcr.io/gofractally/psibase-ubuntu-2004-builder:latest
+FROM ghcr.io/gofractally/psibase-ubuntu-2004-builder:latest as stage1
 
-# Install psibase
-WORKDIR /root
-RUN mkdir psibase
-WORKDIR /root/psibase
+# Install deps
+RUN export DEBIAN_FRONTEND=noninteractive   \
+    && apt-get update                       \
+    && apt-get install -yq                  \
+        wget                                \
+        xz-utils                            \
+        unzip                               \
+        gnupg2                              \
+        curl                                \
+        apt-transport-https                 \
+    && apt-get clean -yq                    \
+    && rm -rf /var/lib/apt/lists/*
+
+WORKDIR /opt
+# Prometheus:
+RUN useradd --no-create-home --shell /bin/false prome
+RUN useradd --no-create-home --shell /bin/false node_exporter
+COPY prometheus/prometheus.yml /etc/prometheus/prometheus.yml
+ARG PROMVERSION=prometheus-2.43.0+stringlabels.linux-amd64
+RUN mkdir -p /var/lib/prometheus    \
+    && cd /var/lib/prometheus       \
+    && wget https://github.com/prometheus/prometheus/releases/download/v2.43.0%2Bstringlabels/${PROMVERSION}.tar.gz \

```

Comment:
Is the `prometheus` package in 20.04 too old?

Reply:
I'm not sure, but the instructions on the github don't mention it:
https://github.com/prometheus/prometheus#install

Reply:
That means that the prometheus developers are probably not the ones maintaining the package. It doesn't necessarily mean anything more than that, though.